GliaWidgets/SecureConversations/Welcome/SecureConversations.WelcomeView.swift

```diff
@@ -543,11 +543,13 @@ extension SecureConversations.WelcomeView {
 
         lazy var textView = UITextView().makeView { textView in
             textView.backgroundColor = .clear
+            textView.adjustsFontForContentSizeCategory = true
```

Comment:
I think there are two things here to think about in terms of lacking consistency:
1. We should use dedicated free function `setFontScalingEnabled` to handle specific cases like [this](https://github.com/salemove/ios-sdk-widgets/blob/52fb8e3b24eec85a98f9f7233e508b35d8d148a0/GliaWidgets/Sources/FontScaling/FontScaling.swift#L81) for example.
2. This should be controlled through dedicated style `isFontScalingEnabled`.

Reply:
fixed [ab8bcde](https://github.com/salemove/ios-sdk-widgets/pull/744/commits/ab8bcde359e057a2f05f6c4d32460e8d7f198011)